src/partners/index.md

```diff
@@ -0,0 +1,36 @@
+---
+partners: true
+title: Vue Partners
+proficiencies_urls:
+  VueJs: https://vuejs.org
+  Laravel: https://laravel.com
+  Node: https://nodejs.org
+partners_list:
+    -
+        name: Vehikl
+        logo: vehikl.png
+        proficiencies:
+            - VueJs
+            - Laravel
+            - Node
+            - Team Augmentation
+        location: Ontario, Canada
+        languages:
+            - English
+        url: www.vehikl.com?utm_source=vue_partners_page
```

Comment:
Just realized we probably want to have two separate URLs, one for display and one for actual linking (it looks cleaner without the campaign tracking params)